test/test.js

```diff
@@ -1,5 +1,6 @@
 var assert = require("assert");
 var absolution = require('../index.js');
+const exp = require("constants");
```

Comment:
Why is this being imported?

Reply:
I will remove it, my vs code automatically imported that.